libs/expo/betterangels/src/lib/screens/Client/Docs/index.tsx

```diff
@@ -39,6 +48,43 @@ export default function Docs({
         isModalVisible={isModalVisible}
         closeModal={() => setModalVisible(false)}
       />
+      <View style={{ gap: Spacings.xs, marginTop: Spacings.sm }}>
+        {client?.clientProfile.docReadyDocuments &&
+          client?.clientProfile.docReadyDocuments?.length > 0 && (
+            <Documents
+              title="Doc-Ready"
+              {...props}
+              data={
+                client?.clientProfile.docReadyDocuments as ClientDocumentType[]
+              }
+              clientId={client?.clientProfile.id}
+            />
+          )}
+        {client?.clientProfile.consentFormDocuments &&
```

Comment:
`consentFormDocuments` should be `formDocuments` everywhere
```suggestion
        {client?.clientProfile.formDocuments &&
```

Reply:
not availabe on backend. lets discuss this tomorrow

Reply:
oh, my bad. I can update this